winterfell/webapp/src/js/claimConfirmCtrl.js

```diff
@@ -1,8 +1,98 @@
 var app = angular.module('vetafiApp');
-app.controller('claimConfirmCtrl', ['$scope', '$state', 'net', 'claimService', '$stateParams',
-  function($scope, $location, net, claimService, $stateParams) {
+app.controller('claimConfirmCtrl', ['$scope', '$state', 'net', 'claimService', '$stateParams', '$uibModal',
+  function($scope, $location, net, claimService, $stateParams, $uibModal) {
     $scope.claimId = $stateParams.claimId;
 
+    $scope.user = {
+      firstName: 'Aaron',
+      lastName: 'Hsu'
+    };
+
+    $scope.vaAddress = {
```

Comment:
done. Was going to do that in a future change (linking actual info instead of placeholder items)